server/lib/nl/fulfillment/existence.py

```diff
@@ -62,21 +63,51 @@ def __init__(self, state: PopulateState, place2keys: Dict):
     # has data for that SV.
     self.existing_svs = {}
 
+  def _get_sv_place_facets(self):
```

Comment:
Maybe add descriptive summary of what the function does, esp the computing the facets with max occurrences?

Reply:
done